src/toffy/image_stitching.py

```diff
@@ -86,8 +86,8 @@ def get_tiled_names(fov_list, run_dir):
         if default_name in fov_list:
             # get tiled name
             tiled_name = fov.get("name")
-            # filter out moly fovs
-            if tiled_name != "MoQC":
+            # filter out non-tiled and moly fovs
+            if re.search(re.compile(r"(R\+?\d+)(C\+?\d+)"), tiled_name) and tiled_name != "MoQC":
```

Comment:
We could refactor the regex searching to a function, since we've been using it quite a lot. What do you think?

Reply:
Here is doesn't actually need the values, just checks for a None return to skip fovs without the row and column specs included. I agree that this would be a useful function though, maybe in alpineer?

Reply:
@camisowers Yeah adding it in alpineer could be good at some point.